build.sh

```diff
@@ -10,6 +10,4 @@ fi
 tag=${tag/tags\//}
 
 dotnet test .\\src\\AggregateRepository.Core.Tests\\AggregateRepository.Core.Tests.csproj
-if %errorlevel% neq 0 exit /b %errorlevel%
```

Comment:
Why did we get rid of this? Wasn't this added to stop it building if the tests failed?

Reply:
That line was causing errors when I ran the script, the set -e at the top already tells it to stop if anything returns a non-zero exit code.